addons/hr/static/src/views/open_chat_hook.js

```diff
@@ -8,6 +8,7 @@ patch(helpers, {
         ...helpers.SUPPORTED_M2X_AVATAR_MODELS,
         "hr.employee",
         "hr.employee.public",
+        "hr.candidate", // Just to avoid the traceback
     ],
     buildOpenChatParams(resModel, id) {
         if (["hr.employee", "hr.employee.public"].includes(resModel)) {
```

Comment:
Since it does not really work with `hr.candidate`, we could do this: 

```suggestion
        if (resModel === "hr.candidate") {
            return;
        } else if (["hr.employee", "hr.employee.public"].includes(resModel)) {
```
By doing that, we will make sure we will not have other issues since we did not manage `hr.candidate` inside `hr_employee_chat` widget. Actually, I think it was a mistake to use that widget in form view of that model. 🤔